src/components/selfServicePage/MemberSelfService.js

```diff
@@ -97,9 +102,9 @@ function ManageAccountMenu() {
         >
           <MenuItem onClick={handleClose}>Profile information</MenuItem>
           <MenuItem onClick={handleClose}>Leave community</MenuItem>
-          <MenuItem onClick={handleClose}>Log out</MenuItem>
+          <MenuItem onClick={firebaseAuthenticator.logOut}>Log out</MenuItem>
```

Comment:
Log out doesn't fully work - I need to refresh the page for it to make effect.

Reply:
Ooof, that was quite a bug, refreshing page mechanism was broken -- because of the asynchronous registering of user status changes, after refresh you were automatically redirected to starting page. Kudos for spotting this out. :)

Reply:
Hm... the bug still seems to be there on 05dee9cc.

Reply:
The bug is fixed now. Thanks!